javascript/javascript2/week1/index2js2week1.html

```diff
@@ -0,0 +1,79 @@
+<head>
+  <title>HyfBay</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <link rel="stylesheet" type="text/css" href="main.css" />
+  <link
+    href="https://fonts.googleapis.com/css?family=Open+Sans:400,800"
+    rel="stylesheet"
+  />
+</head>
+<body>
+  <main>
+    <header>
+      <div class="wrapper">
+        <h1>HyfBay</h1>
+        <h2>- Buy the okay'est products</h2>
+      </div>
+    </header>
+    <section class="table-view-control">
+      <div class="search">
+        <label>What are you looking for?</label>
+        <input
+          type="search"
+          id="searchName"
+          placeholder="Search for a product"
+        />
+        <label>Enter a price limit</label>
+        <input type="number" id="maxPrice" placeholder="Enter max price" />
+      </div>
+
+      <div class="sort">
+        <label>Sort:</label>
+        <select>
+          <option value="Default">Default</option>
+          <option value="LowToHigh">Low to high</option>
+          <option value="HighToLow">High to low</option>
+        </select>
+      </div>
+      <div class="filters">
+        <div class="country">
+          <label>Ships to</label>
+          <select>
+            <option value="denmark">Denmark</option>
+            <option value="sweden">Sweden</option>
+            <option value="norway">Norway</option>
+            <option value="germany">Germany</option>
+            <option value="iceland">Iceland</option>
+            <option value="england">England</option>
+          </select>
+        </div>
+        <div class="price">
+          <input type="range" id="start" name="volume" min="0" max="11" />
+          <label for="volume">Lower Price:</label>
+        </div>
+        <div><button>New Render Products</button></div>
+      </div>
+    </section>
+    <section class="products">
+      <ul>
+        <li>
```

Comment:
I think you only need one list here. This markup is a list nested within another list. 
The `<ul></ul>` is the list element (ul stands for unordered list) and the `<li></li>` is the list item (li). In your markup we have a list on line 58 that has one item (line 59) and in this item we have another list with 4 items in it. One list should be enoough :) You shjould keep the markup between lines 60 and 65 and remove thre wrapping list and listy item.